src/spectrecoff/Output.fs

```diff
@@ -139,4 +139,9 @@ let rec payloadToRenderable (payload: OutputPayload) =
 let toConsole (payload: OutputPayload) =
     payload
     |> payloadToRenderable
-    |> List.iter AnsiConsole.Write
\ No newline at end of file
+    |> List.iter AnsiConsole.Write
+
+type OutputPayload with 
+    member x.toMarkedUpString = toMarkedUpString x
+    member x.toRenderable = payloadToRenderable x 
```

Comment:
Also, might there be a way to define these members directly on the type? I couldn't find one, since they reference those `toSomething` methods defined below, but maybe there is one I don't know of.

Reply:
There is, sure, but as they are intended for external use, I find it cleaner to not have them on the type here, but instead offer them as extensions. Have a look at [this page](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/type-extensions), in particular the section on 'Intrinsic type extensions'.

Reply:
Renamed `x` to `self`, though.